docs/clusterfuzzlite/running_clusterfuzzlite.md

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
Something tells me this approach of describe high-level generic concepts here and then guides on specific systems (e.g. github actions) isn't the right approach. maybe I should just redescribe everything in each guide. WDYT?